docs/manual/index-checker.tex

```diff
@@ -171,6 +175,9 @@ \section{Lower bounds\label{index-lowerbound}}
 \item[\refqualclass{checker/index/qual}{LowerBoundUnknown}]
   There is no information about the value.
   It may not be used as an index for a sequence, because it might be too low.
+\item[\refqualclass{checker/index/qual}{LowerBoundBottom}]
```

Comment:
I would be inclined to put qualifiers that the user does not write at the end of the list (that is, after `@PolyLowerBound`).